tests/unit/amo/components/TestHomeHeroBanner.js

```diff
@@ -33,4 +40,55 @@ describe(__filename, () => {
     expect(carousel).toHaveLength(1);
     expect(carousel).toHaveProp('random', true);
   });
+
+  it('calls tracking on home page view', () => {
+    const fakeTracking = createFakeTracking();
+    const root = shallowRender({ _tracking: fakeTracking });
+
+    const { variant } = root.instance().props;
+
+    sinon.assert.calledWith(fakeTracking.sendEvent, {
+      action: `${AB_HOME_HERO_TEST_NAME} Page View`,
+      category: `AMO ${AB_HOME_HERO_TEST_NAME}_EXPERIMENT: ${variant}`,
+      label: '',
+    });
+
+    sinon.assert.calledOnce(fakeTracking.sendEvent);
+  });
+
+  it('calls tracking on hero click', () => {
+    const fakeTracking = createFakeTracking();
+
+    const root = shallowRender({
+      _tracking: fakeTracking,
+    });
+
+    const { variant } = root.instance().props;
+
+    sinon.assert.calledWith(fakeTracking.sendEvent, {
+      action: `${AB_HOME_HERO_TEST_NAME} Page View`,
+      category: `AMO ${AB_HOME_HERO_TEST_NAME}_EXPERIMENT: ${variant}`,
+      label: '',
+    });
+
+    sinon.assert.calledOnce(fakeTracking.sendEvent);
+
+    const heroBanner = root.find(Hero);
+
+    // We'll use the first item in the home heroes array as an example.
+    const heroItem = heroBanner.prop('sections')[0];
+    const heroLink = shallow(heroItem).find('.HeroSection-link-wrapper');
```

Comment:
Why not using `find()` and `.at(0)`?

Reply:
hey @willdurand, can you clarify where you mean?

Reply:
ahm, nevermind. I did not read the code correctly and missed the line 76...

I am not sure we want to do that though. Can't we just get the `onClick` prop of the first item and assert that?

Reply:
I am trying to simulate the click on the hero's respective link ('.HeroSection-link-wrapper'). It doesn't seem to work without. Is this what you mean? I had tried I think what you're mentioning here earlier too but it didn't work:

```bash
 TypeError: heroItem.simulate is not a function

      77 |     const heroItem = heroBanner.prop('sections')[0];
      78 | 
    > 79 |     heroItem.simulate('click', createEvent());
```


I *think* b/c the onclick is passed down to the inner component..?

the following seems to work? is this any better:

```javascript
const heroItem = heroBanner.prop('sections')[0];
heroItem.props.onClick(); 
```